src/cli_common.c

```diff
@@ -428,19 +428,33 @@ sds cliVersion(void) {
 }
 
 /* This is a wrapper to call redisConnect or redisConnectWithTimeout. */
-redisContext *redisConnectWrapper(const char *ip, int port, const struct timeval tv) {
-    if (tv.tv_sec == 0 && tv.tv_usec == 0) {
-        return redisConnect(ip, port);
-    } else {
-        return redisConnectWithTimeout(ip, port, tv);
+redisContext *redisConnectWrapper(const char *ip, int port, const struct timeval tv, int nonblock) {
```

Comment:
Maybe we could rename it to valkeyConnect or just cliConnect?

Reply:
I heard that valkey will use libvalkey instead of hiredis in the future, replacing symbol at that time together?

Reply:
SGTM.